debian/control

```diff
@@ -30,7 +30,7 @@ Depends:
     python3-taglib,
     xdg-utils
 Recommends:
-    gir1.2-appindicator3-0.1,
+    gir1.2-ayatanaappindicator3-0.1 | gir1.2-appindicator3-0.1,
```

Comment:
Very good, @mathiascode. You're almost ready to become an official Debian developer.

Reply:
Not ready to become a Debian user though. :]